andela/ah-frontend-targaryen
# #161861613 redirect to login page

This PR fixes redirection after signup by redirecting to the login page.
It also improves the wording of some variables and action types

#### How should this be manually tested?

Run `npm install` to install all dependencies
Run the Authors Heaven application `npm start`
Navigate to the signup page and register a user.
On successful signup, there should be redirection to the login page

#### What are the relevant pivotal tracker stories?

[#161861613](https://www.pivotaltracker.com/story/show/161861613)



#### Any background context you want to add?
N/A

## Checklist: I completed these to help reviewers




  No? Unsure? -> configure your IDE, format the code and add the changes with `git add . && git commit --amend`


  No? -> write tests and add them to this commit `git add . && git commit --amend`


  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


  No? Unsure? -> execute command `git pull --rebase origin develop`
